massa-models/src/config/constants.rs

```diff
@@ -117,6 +117,8 @@ pub const DEFERRED_CREDITS_BOOTSTRAP_PART_SIZE: u64 = 100;
 pub const EXECUTED_OPS_BOOTSTRAP_PART_SIZE: u64 = 10;
 /// Maximum number of consensus blocks in a bootstrap batch
 pub const CONSENSUS_BOOTSTRAP_PART_SIZE: u64 = 50;
+/// Maximum number of consensus block ids when sending a bootstrap cursor from the client
+pub const MAX_CONSENSUS_BLOCKS_IDS: u64 = 300;
```

Comment:
How was this chosen ?

Reply:
ping

Reply:
cc @Eitu33

Reply:
Sorry for the delay I missed it, it was chosen at the consensus bootstrap streaming testnet a while back

Reply:
I don't know why it appears as an added change here tho

Reply:
Ah no nevermind it was fix I made at the beginning of the month, I forgot about it

Reply:
It only concerns the bootstrap retry

Reply:
ok